.github/workflows/build-release.yml

```
@@ -174,11 +174,11 @@ jobs:
           CA_PACKAGE: bfd-server-war
           CA_DATA_DICTIONARY_ASSETS: |
             [
-              "bfd-server-war-V1-data-dictionary-${{ inputs.releaseVersion }}.csv",
-              "bfd-server-war-V2-data-dictionary-${{ inputs.releaseVersion }}.csv",
-              "bfd-server-war-V1-data-dictionary-${{ inputs.releaseVersion }}.json",
-              "bfd-server-war-V2-data-dictionary-${{ inputs.releaseVersion }}.json",
-              "bfd-server-war-data-dictionary-${{ inputs.releaseVersion }}.xlsx"
+              "bfd-server-war-${{ inputs.releaseVersion }}-v1-data-dictionary.csv",
+              "bfd-server-war-${{ inputs.releaseVersion }}-v2-data-dictionary.csv",
+              "bfd-server-war-${{ inputs.releaseVersion }}-v1-data-dictionary.json",
+              "bfd-server-war-${{ inputs.releaseVersion }}-v2-data-dictionary.json",
+              "bfd-server-war-${{ inputs.releaseVersion }}-data-dictionary.xlsx"
```

I think I may have made a mistake here, @malessi... this probably ought to be reverted, right? I'm not quite sure what compelled me to update these... Hmm.
```suggestion
              "bfd-server-war-${{ inputs.releaseVersion }}-v1-data-dictionary.csv",
              "bfd-server-war-${{ inputs.releaseVersion }}-v2-data-dictionary.csv",
              "bfd-server-war-${{ inputs.releaseVersion }}-v1-data-dictionary.json",
              "bfd-server-war-${{ inputs.releaseVersion }}-v2-data-dictionary.json",
              "bfd-server-war-${{ inputs.releaseVersion }}-data-dictionary.xlsx"
```

---

Yeah, I believe so. Running `mvn verify` in `bfd-server-docs` generates files with named in convention with the original scheme, not this updated scheme.

---

Reverting the revert... the maven-stored artifacts in CodeArtifact differ from what's produced in the dist directory. We'll have to follow up on this one in BFD-3121.